docs/architecture/architecture-and-security.md

```diff
@@ -73,29 +78,195 @@ The following image gives a more detailed overview how Landscaper instances are
-of a component which creates
+[Installation](https://github.com/gardener/landscaper/blob/master/docs/usage/Installations.md) for a Landscaper Instance.
 
-- a new shoot custom resource in the Garden-Resource-Cluster-Project for which the Gardener creates the 
-  Resource-Shoot-Cluster
-- a new namespace on one of the Target-Shoot-Clusters for a new Landscaper instance
-- a Landscaper deployment on the Target-Shoot-Cluster in the newly created namespace
+The LaaS furthermore creates two [Targets](https://github.com/gardener/landscaper/blob/master/docs/usage/Targets.md)
+as input for the Installation containing the credentials
 
-The LaaS furthermore creates two [Targets](https://github.com/gardener/landscaper/blob/master/docs/usage/Targets.md) 
-as input for the Installation containing the credentials 
+- for a Target-Shoot-Cluster. These credentials are fetched from one of the ServiceTargetConfig custom resources and 
+  are used to install all required artefacts for the Landscaper Instance on a Target-Shoot-Cluster.
 
-- for the Garden project Garden-Resource-Cluster-Project (for shoot resource creation)
+- for the Garden project Garden-Resource-Cluster-Project to create a Resource-Shoot-Cluster on which the customer 
+  can later deploy its Landscaper resources like Installations, Targets, etc.
 
-- for a Target-Shoot-Cluster. These credentials are fetched from one of the ServiceTargetConfig custom resources. 
+The Central Landscaper watches for the Installations and executes them.
 
-The Central Landscaper watches for the Installations and executes them. 
+### Landscaper Instance Details
 
-The installed Landscaper instance watches the just created Resource-Shoot-Cluster on which the customer/user could 
-deploy its own Installations.
+A Landscaper Instance is defined by [this](../../.landscaper/landscaper-instance) component. The component 
+contains a [blueprint](../../.landscaper/landscaper-instance/blueprint/installation/blueprint.yaml) with five 
+[sub installations](https://github.com/gardener/landscaper/blob/master/docs/usage/Blueprints.md#nested-installations), 
+which are deploying:
+
+- **[shoot](../../.landscaper/landscaper-instance/blueprint/installation/shoot-cluster-subinst.yaml)**: 
+  A new shoot custom resource in the Garden-Resource-Cluster-Project for which the Gardener creates a new 
+  Resource-Shoot-Cluster
+- **[landscaper](../../.landscaper/landscaper-instance/blueprint/installation/landscaper-deployment-subinst.yaml)**: 
+  A new Landscaper and its deployers in new namespace on one of the Target-Shoot-Clusters. This Landscaper is watching 
+  and processing the Landscaper resources on the Resource-Shoot-Cluster.
+- **[ls-service-target-shoot-sidecar-server](../../.landscaper/landscaper-instance/blueprint/installation/sidecar-subinst.yaml)**: 
+  Two controllers handling the access rights of the customer/users to the Resource-Shoot-Cluster as well as the 
```

Comment:
Two controller on the Target-Shoot-Cluster

Reply:
I have changed it